Library/Homebrew/env_config.rb

```diff
@@ -132,9 +132,10 @@ module EnvConfig
         boolean:     true,
       },
       HOMEBREW_CURLRC:                           {
-        description: "If set, do not pass `--disable` when invoking `curl`(1), which disables the " \
-                     "use of `curlrc`.",
-        boolean:     true,
+        description: "If set to a full path (i.e. beginning with `/`), pass it with `--config` when invoking " \
```

Comment:
```suggestion
        description: "If set to an absolute path (i.e. beginning with `/`), pass it with `--config` when invoking " \
```
Maybe? "absolute" strikes me as more standard terminology, but not a strong opinion.